src/plugins/workspace/public/components/utils/feature.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const isFeatureDependOnSelectedFeatures = (
```

Comment:
It seems this function is checking if `featureId` is depend by selected features? So we want to know if selected `selectedFeatureIds`, should `featureId` been selected. Am I right?

Reply:
You're right. I think we can rename `isFeatureDependOnSelectedFeatures` to `isFeatureDependBySelectedFeatures`